plugins/grid-bookmark/src/GridBookmarkWidget/components/ImportBookmarks.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react'
+import { observer } from 'mobx-react'
+import { getSession } from '@jbrowse/core/util'
+import AssemblySelector from '@jbrowse/core/ui/AssemblySelector'
+import { FileLocation } from '@jbrowse/core/util/types'
+import { FileSelector } from '@jbrowse/core/ui'
+import { openLocation } from '@jbrowse/core/util/io'
+import {
+  IconButton,
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Grid,
+  Select,
+  MenuItem,
+  makeStyles,
+  Typography,
+} from '@material-ui/core'
+import CloseIcon from '@material-ui/icons/Close'
+import ImportIcon from '@material-ui/icons/Publish'
+import { GridBookmarkModel } from '../model'
+
+const useStyles = makeStyles(() => ({
```

Comment:
My scrollbar is showing up for the form even though it shouldn't be necessary. Maybe something to do with the margin? Could be good to tinker with to avoid.

Reply:
Found this too, added a fix